packages/core/src/core.jsx

```diff
@@ -59,6 +59,22 @@ export default (
     return class extends Component {
       static displayName = displayName
 
+      static getDerivedStateFromProps(props, state) {
```

Comment:
What is the added value compared to using a constructor ?

Reply:
This function (lifecycle) is called before each rerender (when props or state change)

Reply:
each time we receive a prop update, we wanted (but maybe this is dumb?) to remove the prop that is used to load data